env.default

```diff
@@ -0,0 +1,15 @@
+# Wagtail config
+
+ALLOWED_HOSTS=*
+ASSET_DOMAIN=donate.mofoprod.net
+CONTENT_TYPE_NO_SNIFF=True
+CORS_WHITELIST=*
+CORS_REGEX_WHITELIST=""
+DATABASE_URL=""
+DEBUG=True
+DJANGO_SECRET_KEY=secret
+SET_HSTS=False
+SSL_REDIRECT=False
+USE_S3=False
+X_FRAME_OPTIONS=DENY
+XSS_PROTECTION=True
```

Comment:
@solarissmoke Could I suggest adding `REDIS_URL=""` here?

Reply:
Could you explain why?

Reply:
Just to make things a little easier to find. It's quite possible that a developer might want to install and use redis locally to get a more 'production-like' stack. If they did, it's slightly clearer what they need to update in their local `.env` file to make that happen.